<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ldmt_rope</class>
 <widget class="QMainWindow" name="ldmt_rope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rope</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(55,55,55);
}
*{
color:rgb(175,175,175);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{background-color: rgb(55,55,55);}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QRadioButton" name="btn_normalRope">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string>Generate rope with natural topo.</string>
      </property>
      <property name="text">
       <string>Normal Rope</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="btn_helixRope">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string>Generate rope with perfect loop.</string>
      </property>
      <property name="text">
       <string>Helix Rope</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="btn_generate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Generate rope from curves.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(68, 145, 185, 200), stop:1 rgba(108, 185, 224, 180));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(88, 165, 205, 255), stop:1 rgba(128, 205, 244, 255));
color:rgb(25,25,25);
text-decoration: underline!important;}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(108, 185, 224, 180), stop:1 rgba(68, 145, 185, 200));
color:rgb(25,25,25);}</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="text_strandsCount">
      <property name="statusTip">
       <string>Input the strands count of rope.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 4px;
border:1px solid rgb(30,30,30);
background-color: rgb(30,30,30, 215);
color:rgb(175,175,175);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Strands Count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{

}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
